### PR TITLE
Topological vector spaces over the reals are contractible

### PR DESCRIPTION
**Pull request by GeoffreySangston**
This could be an alternative proof for S176|P199 and S30|P199, from https://github.com/pi-base/data/pull/886#issue-2645410914

Edit: I should assert that it is a real topological vector space.

Edit: Added S21|P199 (weak topology on separable Hilbert space).

### CODE REVIEW

**Comment by GeoffreySangston** [comment]
Actually, I may add this as a reference [normed vector space => topological vector space](https://math.stackexchange.com/questions/167890/proof-that-every-normed-vector-space-is-a-topological-vector-space)

**Comment by prabau** [comment]
I agree with this change.  But note that S106 is in the process of being merged with S30. See #863. 
So maybe this can wait until after that, and then applied to S30 instead.

**Comment by yhx-12243** [comment]
In fact the pull request of adding P199 to S30 is already in #886, so this pull request will automatically closed once #863 is resolved successfully.

**Comment by GeoffreySangston** [comment]
> I agree with this change. But note that S106 is in the process of being merged with S30. See #863. So maybe this can wait until after that, and then applied to S30 instead.

Ah I missed this somehow. Maybe we should just close this PR then and add S21|P199 after.

**Comment by prabau** [comment]
You can mark this one as draft for now and then revisit after #863 is merged.

File: spaces/S000106/properties/P000199.md
Comment (yhx-12243) [inline spaces/S000106/properties/P000199.md:7]:
```suggestion
```

**Comment by prabau** [comment]
Do you want to do an associated cleanup for traits that are derivable from Contractible here, or leave that for later?

**Comment by GeoffreySangston** [comment]
> Do you want to do an associated cleanup for traits that are derivable from Contractible here, or leave that for later?

I could do either, but later may be later than is desirable.

**Comment by prabau** [comment]
Yeah, when adding new traits, we usually do associated cleanup at the same time, so it shows a tidier minimal set of traits.  Sometimes, but totally optional, even a little more (e.g., the space being not Indiscrete).

**Comment by GeoffreySangston** [comment]
I've converted to a draft until I find time to do the cleanup.

**Comment by prabau** [comment]
> Is there any reason to keep S30|P53 (metrizable) and S30|P55 (completely metrizable)?

No reason.  Feel free to clean up anything obvious or more.

**Comment by GeoffreySangston** [comment]
Looks like completely metrizable + pseudocompact makes $\sigma$-compact redundant. 

[π-Base, Search for `completely metrizable + ~$\sigma$-compact + Pseudocompact`](https://topology.pi-base.org/spaces?q=completely+metrizable+%2B+%7E%24%5Csigma%24-compact+%2B+Pseudocompact)

**Comment by GeoffreySangston** [comment]
I think that's a minimal set of traits. Just making a separate comment because there appears to be a website bug. The long formula
[π-Base, Search for `~sigma-compact + pseudocompact + weakly locally compact + separable + locally arc connected + completely metrizable + has a group topology + indiscrete + Contractible`](https://topology.pi-base.org/spaces?q=%7Esigma-compact+%2B+pseudocompact+%2B+weakly+locally+compact+%2B+separable+%2B+locally+arc+connected+%2B+completely+metrizable+%2B+has+a+group+topology+%2B+indiscrete+%2B+Contractible)
makes the website think I've asserted 'Compact'.

**Comment by GeoffreySangston** [comment]
Resetting...

The following says that pseudocompact = false is redundant. Going to hold off on deleting it so I can double check later.
[π-Base, Search for `pseudocompact + ~weakly locally compact + completely metrizable  + ~Indiscrete`](https://topology.pi-base.org/spaces?q=pseudocompact+%2B+%7Eweakly+locally+compact+%2B+completely+metrizable++%2B+%7EIndiscrete)

**Comment by prabau** [comment]
Approving as is.

Even this simpler search [π-Base, Search for `metrizable + ~WLC + pseudocompact`](https://topology.pi-base.org/spaces?q=metrizable+%2B+%7EWLC+%2B+pseudocompact)
to derive [Metrizable + not weakly locally compact ==> not pseudocompact] is really a bear.

We can leave it for now, but I wish there was an easier derivation.  @StevenClontz Any comment?

I also wonder how to revert a commit from github.dev.

**Review by prabau — APPROVED** [review]

**Pull request merged**
